src/Workspaces/Core/Portable/Storage/SQLite/v2/SQLitePersistentStorage.cs

```diff
@@ -189,6 +190,8 @@ private void Initialize(SqlConnection connection, CancellationToken cancellation
             EnsureTables(connection, Database.Main);
             EnsureTables(connection, Database.WriteCache);
 
+            LoadExistingStringIds(connection);
```

Comment:
What's this change for?

Reply:
Will doc.  But something i noticed while testing.  we were hitting the DB heavily on the initial loads reading in all the string->id pairs, one at a time as different reads hit different strings.

This is much more efficient as we can just stream them all in in one swell foop at initialization time.

Reply:
but i can leave this out of this PR :)

Reply:
removed.  will add in followup.

Reply:
Please do a separate pr 🙏 will make it easier to diagnose any perf issues

Reply:
yup yup.  it's removed! :)